src/common/vitals/vital-metric.js

```diff
@@ -20,7 +20,7 @@ export class VitalMetric {
     this.history.push(state)
     this.#subscribers.forEach(cb => {
       try {
-        cb(this.current)
+        cb(state)
```

Comment:
Where was `this.current` even defined?

Reply:
Right below in this file as a getter, so it basically just go and grab the state defined here again

Reply:
Ah ok 👍